lib/faker/address/es.ex

```diff
@@ -0,0 +1,734 @@
+    "Kazajistan",
+    "Kenia",
+    "Kiribati",
+    "Corea",
+    "Kuwait",
+    "Letonia",
+    "Líbano",
+    "Liberia",
+    "Liechtenstein",
+    "Lituania",
+    "Luxemburgo",
+    "Macao",
+    "Macedonia",
+    "Madagascar",
+    "Malawi",
+    "Malasia",
+    "Maldivas",
+    "Mali",
+    "Malta",
+    "Martinica",
+    "Mauritania",
+    "Méjico",
+    "Micronesia",
+    "Moldavia",
+    "Mónaco",
+    "Mongolia",
+    "Montenegro",
+    "Montserrat",
+    "Marruecos",
+    "Mozambique",
+    "Namibia",
+    "Nauru",
+    "Nepal",
+    "Holanda",
+    "Nueva Zelanda",
+    "Nicaragua",
+    "Niger",
+    "Nigeria",
+    "Noruega",
+    "Omán",
+    "Pakistan",
+    "Panamá",
+    "Papúa Nueva Guinea",
+    "Paraguay",
+    "Perú",
+    "Filipinas",
+    "Poland",
+    "Portugal",
+    "Puerto Rico",
+    "Rusia",
+    "Ruanda",
+    "Samoa",
+    "San Marino",
+    "Santo Tomé y Principe",
+    "Arabia Saudí",
+    "Senegal",
+    "Serbia",
+    "Seychelles",
+    "Sierra Leona",
+    "Singapur",
+    "Eslovaquia",
+    "Eslovenia",
+    "Somalia",
+    "España",
+    "Sri Lanka",
+    "Sudán",
+    "Suriname",
+    "Suecia",
+    "Suiza",
+    "Siria",
+    "Taiwan",
+    "Tajikistan",
+    "Tanzania",
+    "Tailandia",
+    "Timor-Leste",
+    "Togo",
+    "Tonga",
+    "Trinidad y Tobago",
+    "Tunez",
+    "Turquia",
+    "Uganda",
+    "Ucrania",
+    "Emiratos Árabes Unidos",
+    "Reino Unido",
+    "Estados Unidos de América",
+    "Uruguay",
+    "Uzbekistan",
+    "Vanuatu",
+    "Venezuela",
+    "Vietnam",
+    "Yemen",
+    "Zambia",
+    "Zimbabwe"
+  ])
+
+  @doc """
+  Return country code.
+  """
+  @spec country_code() :: String.t()
+  sampler :country_code, ["ES"]
+
+  @doc """
+  Returns a geohash.
+  """
+  @spec geohash() :: binary
+  def geohash do
+    bits = encode_to_bits(latitude(), longitude(), Faker.random_between(5, 25) * 5)
+
+    to_geobase32(bits)
+  end
+
+  defp encode_to_bits(lat, lon, bits_length) do
+    starting_position = bits_length - 1
+    lat_bits = lat_to_bits(lat, starting_position - 1) # odd bits
+    lon_bits = lon_to_bits(lon, starting_position) # even bits
+    geo_bits = lat_bits + lon_bits
+    <<geo_bits::size(bits_length)>>
+  end
+
+  defp to_geobase32(bits) do
+    chars = for << c::5 <- bits >>, do: Enum.fetch!(@geobase32, c)
+    chars |> to_string
+  end
+
+  defp lon_to_bits(lon, position) do
+    geo_to_bits(lon, position, {-180.0, 180.0})
+  end
+
+  defp lat_to_bits(lat, position) do
+    geo_to_bits(lat, position, {-90.0, 90.0})
+  end
+
+  defp geo_to_bits(_, position, _) when position < 0 do
+    0
+  end
+  defp geo_to_bits(n, position, {gmin, gmax}) do
+    mid = (gmin + gmax) / 2
+
+    if n >= mid do
+        round(:math.pow(2, position)) + geo_to_bits(n, position - 2, {mid, gmax})
+    else
+        geo_to_bits(n, position - 2, {gmin, mid})
+    end
+  end
+
+  @doc """
+  Return random latitude.
+  """
+  @spec latitude() :: float()
+  def latitude do
```

Comment:
same thing here, I really hope that `latitude` is the same thing in all countries :)

Will not comment in other functions like this, please check them too.